cobalt/black_box_tests/black_box_cobalt_runner.py

```diff
@@ -86,7 +86,8 @@ def PollUntilFoundOrTestsFailedWithReconnects(self, css_selector):
               selenium_exceptions.NoSuchElementException,
               selenium_exceptions.NoSuchWindowException,
               selenium_exceptions.WebDriverException) as e:
-        # If the page
+        # If the page crashes or the browser is disconnected, the webdriver 
```

Comment:
Oh, what I meant to say her that if the page is reloaded, the webdriver client status becomes stale and should be reconnected.

Specifically, any previous element ID that was returned is now stale, and for the black box that that happens for the element that signals the test status.